### DISCUSSION TITLE
Possible to change parameters on deye inverters?

### DISCUSSION THREAD

**Discussion by torsteinelv** [StephanJoubert/home_assistant_solarman#124] [General]
Is it possible to change schedule and config on the inverter?

**Comment by linuxdep** [comment]
i have the same Question

**Comment by GyroGearl00se** [comment]
Regarding parameters following yml worked for me:

https://github.com/StephanJoubert/home_assistant_solarman/issues/88#issuecomment-1339941709

This worked for me to get correct values out of a 10k deyd hybrid.

**Reply by linuxdep** [reply]
> Regarding parameters following yml worked for me:
> 
> [#88 (comment)](https://github.com/StephanJoubert/home_assistant_solarman/issues/88#issuecomment-1339941709)
> 
> This worked for me to get correct values out of a 10k deyd hybrid.

This was not the question! Reading is ok, but he ask to write some parameter back to inverter.

**Reply by GyroGearl00se** [reply]
Sorry my bad. I was focused on reading data in different discussion/issues that I must have got it wrong.

From what I can see it looks like currently it is not possible to write data.

**Comment by adnovea** [comment]
+1

I have the same requirement. I really appreciate this module and I'd like to be able to have some controls over my Deye Sun 5K.
It will be very nice if there is in next releases the possibility to send some commands to the inverter.

May thanks for those involved to keep this project updated.
